rpcs3/CMakeLists.txt

```diff
@@ -124,7 +128,7 @@ if(NOT MSVC)
 
 	add_compile_options(-msse -msse2 -mcx16 -mrtm)
 
-	if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+	if(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
```

Comment:
This is a case of double expansion, drop the ${} around the variable name.

Reply:
Indeed, thank you for the review